src/dialogs/image-dialog.js

```diff
@@ -0,0 +1,5 @@
+const { BaseDialog } = require('botfuel-dialog');
+
+class ImageDialog extends BaseDialog {}
```

Comment:
Maybe FileDialog will be more explicit. You trigger this one when you receive a file, not necessary a image

Reply:
@Benjamin15 @JosselinMorau maybe there is two dialogs to implement, not only ImageDialog but also an AttachmentDialog or as you said @Benjamin15 a FileDialog.
We need to validate if we want to have different dialogs for images and others attachments types a user can send to the bot or use only one dialog for each attachment types.